parquet/src/column/reader.rs

```diff
@@ -212,14 +212,16 @@ where
         Ok((values, levels))
     }
 
-    /// Read up to `num_records` returning the number of complete records, non-null
-    /// values and levels decoded
+    /// Read up to `num_records` whole records, returning the number of complete
+    /// records, non-null values and levels decoded. Records will not be partially read
```

Comment:
What does "partially read" mean? Like does it mean that the a record will not be read unless there is sufficient data for all columns?

If so, maybe

```suggestion
    /// records, non-null values and levels decoded. Records will not be partially read, 
    /// they are only read as long as there is sufficient data for all columns available. 
```

Reply:
You only have a single column here, this is referring to record shredding of repeated values, will clarify further...